ONTOLOGY_GUIDE.md

```diff
@@ -297,23 +313,26 @@ These tests are run every time a change is introduced via a pull request in GitH
 ```
           
 ## Candidate Tests for Quality Compliance
-In addition to the tests listed above, the FIBO leadership team is contemplating implementing the following tests (through automation where possible, but in review processes at a minimum), to further ensure the quality of the ontologies we publish.  These include:
+In addition to the tests listed above, the FIBO Community is contemplating implementing the following tests (through automation where possible, but in review processes at a minimum), to further ensure the quality of the ontologies we publish.  These include:
 
 ### T1. Polysemous elements
 
 An ontology element whose name has different meanings is included in the ontology to represent more than one conceptual idea. For example, the class “Theatre” is used to represent both the artistic discipline and the place in which a play is performed.
 
-The FIBO policy for treating polysemes is to create separate concepts for each polyseme and differentiate them via (1) different names (avoiding numeric extensions on the same name in favor of longer or more accurate names), and (2) different definitions, including distinguishing restrictions, to the degree possible.
+The FIBO policy for treating polysemes is to create separate concepts for each polyseme and differentiate them via 
+
+1. different names (avoiding numeric extensions on the same name in favor of longer or more accurate names), and 
+2. different definitions, including distinguishing restrictions, to the degree possible.
 
 The only case where names are allowed to be duplicated (i.e, where the name used for a concept in one namespace is used for a different concept in another namespace) is for extension purposes, i.e., a concept in a primary FIBO ontology may be extended in a namespace that imports it, either in the original namespace or in a subordinate namespace as appropriate.  Otherwise, FIBO users should assume that even though OWL does not impose a unique names assumption, FIBO policy does.  
```

Comment:
In this case there is only one namespace for the element but different ontologies for the triples.